### PR TITLE
fix(): links to be underlined and coloured and missing Ensono Stacks …

### PR DESCRIPTION
**Pull request by isaac-shek**
#### 📲 What
links to be underlined and coloured and missing Ensono Stacks

#### 🤔 Why
		
Why it's needed, background context.
		
#### 🛠 How
		
More in-depth discussion of the change or implementation.

#### 👀 Evidence
		
Screenshots / external resources / links / etc.
Link to documentation updated with changes impacted in the PR.
		 
#### 🕵️ How to test

Notes on how a reviewer can test the changes, e.g. how to run the tests.

#### ✅ Acceptance criteria Checklist

- [ ] Code peer reviewed?
- [ ] Documentation has been updated to reflect the changes?
- [ ] Passing all automated tests, including a successful deployment?
- [ ] Passing any exploratory testing?
- [ ] Rebased/merged with latest changes from development and re-tested?
- [ ] Meeting the Coding Standards?

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ keywords:
 
 ## What is Ensono Stacks
 
-Stacks is an Ensono open-source project that automates the setup of software development projects. Stacks is for software engineers of all kinds who want to create projects quickly.
+Ensono Stacks is an Ensono open-source project that automates the setup of software development projects. Ensono Stacks is for software engineers of all kinds who want to create projects quickly.
 
 Setting up the infrastructure and frameworks necessary to deliver a software project takes time. A team might spend the first two weeks (sprint 0) making technical decisions, provisioning and configuring environments, pipelines, compute, test frameworks, and more. Stacks automates this setup using infrastructure-as-code to shrink your project's sprint 0 from two weeks to an hour or less.
```
Comment (MarkComerford) [inline docs/index.md:19]: There is a reference to just 'Stacks' on its own in this section, where it says 'Stacks automates'

Comment (MarkComerford) [reply docs/index.md:19]: Also a reference to just stacks on line 33.

**Review by MarkComerford — CHANGES_REQUESTED** [review]

**Review by leesaxby — APPROVED** [review]

**Pull request merged**